feature-libs/user/account/components/login-form/login-form.component.html

```diff
@@ -58,6 +66,14 @@
     ></cx-form-errors>
   </label>
 
+  <input
+    *ngIf="!!form.get('csrf') && (csrf$ | async) as csrf"
+    formControlName="csrf"
+    type="hidden"
+    [name]="csrf.parameterName"
+    [attr.data-csrf]="csrf.token"
+  />
```

Comment:
Let's use the functionality to handle template parts that depend on feature toggle:
 ```suggestion
   <ng-container *cxFeature="'authorizationCodeFlowByDefault'">
    <input
      *ngIf="csrf$ | async as csrf"
      formControlName="csrf"
      type="hidden"
      [name]="csrf.parameterName"
      [attr.data-csrf]="csrf.token"
    />
  </ng-container>
```